### PR TITLE
feat: add possibility to send cards

### PR DESCRIPTION
**Pull request by antoinerabany**
Hi,

I just wanted to add possibility to send cards for nicer display.

https://developers.google.com/hangouts/chat/reference/message-formats/cards

I'll add an example of a working card template as soon I have a working one. 

Thanks again.

### CODE REVIEW

**Comment by julb** [comment]
Hello @antoinerabany, that's a very interesting contribution, thanks !
I've done some remarks in your PR.

The main feedback is (I suppose): if `use_cards=true`, the template `render` will output `json` but as a `string`. In such case, we need to add a `json.loads(...)` to transform the `text_alert` value to a `dict`, otherwise the call to Google Chat will fail.

Could you as well edit the README.md to add the flag ?

Thanks !

File: alertmanager_gchat_integration/app.py [hunk 1]
```diff
@@ -70,10 +70,11 @@ def post_alerts():
         text_alert = J2_TEMPLATE_ENGINE.render(render_payload)
 
         # Post request.
+        message_type = 'cards' if CONFIG['app']['notification'].get('use_cards') else 'text'
         post_request_details = requests.post(
             notification_url,
             json={
-                'text': text_alert
+                message_type: text_alert
```
Comment (julb) [inline alertmanager_gchat_integration/app.py]: if `CONFIG['app']['notification'].get('use_cards')` is true, `text_alert` needs to be converted to a `dict` using `json.loads(text_alert)`.

File: alertmanager_gchat_integration/app.py [hunk 2]
```diff
@@ -70,10 +70,11 @@ def post_alerts():
         text_alert = J2_TEMPLATE_ENGINE.render(render_payload)
```
Comment (julb) [inline alertmanager_gchat_integration/app.py]: Consider maybe to rename the `text_alert` variable to `rendered_alert` to avoid confusion.

**Review by julb — CHANGES_REQUESTED** [review]

**Comment by antoinerabany** [comment]
Hello,

Thanks for your feedback, I implemented your changes.

**Comment by julb** [comment]
Thanks a lot @antoinerabany 👍
I will look at it by tomorrow and gets it released if everything is OK.

**Review by julb — APPROVED** [review]

**Comment by julb** [comment]
Project is released, as well as the associated [Helm Charts](https://github.com/julb/helm-charts/tree/main/julb/alertmanager-gchat-integration).
Thanks for your contribution 👍

**Pull request merged**